src/controllers/user.login.ts

```diff
@@ -0,0 +1,48 @@
+import { Request, Response } from "express"
+import LoginRegisterRepository from "../repositories/loginRegister.repository";
+import { errorHandler } from "../utils/errorHandler";
+import bcrypt from 'bcrypt';
+import { signJwt } from "../utils/authentication";
+
+export const loginUser = async (req: Request, res: Response) => {
```

Comment:
And I think you don't have to create new file for EVERY endpoint, you just need to group them by topic. Login and Register are user endpoints and can be grouped in single controller file.